src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -19501,7 +19433,7 @@ static void Main()
                         var x = F([1, 2, 3]);
                         x.Report();
                     }
-                    static List<T> F<T>(T[] items) => [..items];
+                    static List<T> F<T>(T[] items) => [..items, ..items];
```

Comment:
This test for missing members doesn't make sense with new codegen, thus modified